huaweicloud/services/nat/resource_huaweicloud_nat_dnat_rule.go

```diff
@@ -48,9 +48,17 @@ func ResourcePublicDnatRule() *schema.Resource {
 				Description: "The region where the DNAT rule is located.",
 			},
 			"floating_ip_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
If the field can be set to empty, remove the Computed configuration.

Reply:
done